### ISSUE TITLE
Better logging

### ISSUE THREAD

**Issue by justinmchase** [github/safe-settings#451]
## Prerequisites:

Not related to the UI or API

## New Feature

The logging in the success case is pretty poor. It will generate a lot with information about the webhook request but none of the data in the body is included int he log. As a consequence I cannot see which repo it is operating against or which event is triggering the event and I have no idea what settings its resolving and atteampting to apply which makes debugging things difficult when they seem to not work.

* I would like to know which event is being fired for every webhook call
* I would like to know which repositories are going to be updated when a suborg file is updated (suborg resolution set)
* I would like a log showing which settings are being set to which repo before they are set.
* When a repo has one of its settings set I'd like to know how it resolved those settings, did it come from the settings.yml? A suborg file? Which suborg file? A repository file? What were the values it rejected?

**Comment by svg153** [comment]
Anything related to this https://github.com/github/safe-settings/pull/413? Probably not all, but it would be nice to update this issue with the missing stuff once that PR is merged.

**Comment by HagegeR** [comment]
I completely agree with this request. Even with the latest version, we experience a total absence of info logs after the repository name validation. Enabling debug logs significantly increases the runtime without making the logs easier to follow.

For example, I encountered this issue with the debug log:
> DEBUG (probot): There are no changes for Autolinks for repo [object Object]. Skipping changes
https://github.com/github/safe-settings/blob/1fff61ceb5175ffbf882972a58b5b5a678602c5e/lib/plugins/diffable.js#L95

The repository name should be included here, and this message could potentially be at the info level, indicating either "Skipping changes" or "Applying requested Autolinks."

Another example in the same function we receive a similar "is [object Object]" instead of the repo name:
https://github.com/github/safe-settings/blob/1fff61ceb5175ffbf882972a58b5b5a678602c5e/lib/plugins/diffable.js#L93


Additionally, there is insufficient indication of the sync progress during a full sync, I would like to get a sense of the percentage of the progress if possible.

**Comment by tylerohlsen** [comment]
I'd also love to see an overall success or failure when sync is complete.  That would help with alerting.  And it would help with alerting on the absence of that message to know that the sync didn't happen.
